test/test_gradio_component_cli.py

```diff
@@ -60,7 +60,7 @@ def test_overwrite_deletes_previous_content(tmp_path):
     _create("MyGallery", tmp_path, template="Gallery", overwrite=True)
     _create("MySlider", tmp_path, template="Slider", overwrite=True)
     assert (tmp_path / "frontend" / "interactive" / "InteractiveSlider.svelte").exists()
-    assert not (tmp_path / "frontend" / "static" / "StaticGallery.svelte").exists()
+    assert not (tmp_path / "frontend" / "shared" / "Gallery.svelte").exists()
```

Comment:
Not related to this PR right?

Reply:
I did this cause a test was failing after the front-end reorg.Not sure if it actually fixed the problem though.